### PR TITLE
SW-15984 - New Article: Captcha Plugin

### PR DESCRIPTION
**Pull request by 3stadt**
Shows how to write a CAPTCHA as plugin for the new CAPTCHA implementation coming with Shopware 5.3 (presumably).

### CODE REVIEW

File: exampleplugins/5.2/SwagReCaptcha/ReCaptcha.php [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        if ($response->getStatusCode() !== 200) {
+            return false;
+        }
+
+        $body = $response->getBody();
+        $content = json_decode($body->getContents(), true);
+
+        return is_array($content) &&
+        array_key_exists('success', $content) &&
+        $content['success'] === true ? true : false;
```
Comment (3stadt) [inline exampleplugins/5.2/SwagReCaptcha/ReCaptcha.php]: .done

**Comment by mitelg** [comment]
@3stadt is this ready to merge?

**Comment by 3stadt** [comment]
@mitelg it is.

File: source/developers-guide/implementing-your-own-captcha/index.md [hunk 2]
```diff
@@ -0,0 +1,283 @@
+---
+layout: default
+title: Implementing your own captcha
+github_link: developers-guide/implementing-your-own-captcha/index.md
+tags:
+  - captcha
+  - recaptcha
+  - example
+indexed: true
```
Comment (bcremer) [inline source/developers-guide/implementing-your-own-captcha/index.md]: Please add `shopware_version: 5.3.0` here.

**Comment by bcremer** [comment]
@3stadt @mitelg one blocker before this can be merged: https://github.com/shopware/devdocs/pull/369/files#r95959860

File: source/developers-guide/implementing-your-own-captcha/index.md (hunk 2 above)
Comment (3stadt) [inline source/developers-guide/implementing-your-own-captcha/index.md]: done

**Comment by bcremer** [comment]
:+1:

**Pull request merged**